File: Developer/DeveloperTools/YJTimeProfiler/YJTimeProfiler/ViewController.m
//
//  ViewController.m
//  YJTimeProfiler
//
//  Created by admin on 2017/1/11.
//  Copyright © 2017年 YJCocoa. All rights reserved.
//

#import "ViewController.h"
#import "YJTimeProfiler.h"

@interface ViewController ()

@end

@implementation ViewController

- (void)viewDidLoad {
    [super viewDidLoad];
    YJTimeProfiler.shared.start = YES;
}

- (void)touchesBegan:(NSSet<UITouch *> *)touches withEvent:(UIEvent *)event {
    [super touchesBegan:touches withEvent:event];
    NSLog(@"1");
    for (int i = 0; i < 1000; i ++) {
        NSLog(@"......%d", i);
    }
}

@end

File: Demo/Swift/DeveloperTools/YJTimeProfiler/YJTimeProfiler/ViewController.swift
//
//  ViewController.swift
//  YJLeaks
//
//  Created by 阳君 on 2019/6/17.
//  Copyright © 2019 YJCocoa. All rights reserved.
//

import UIKit

class ViewController: UIViewController {
    
    override func viewDidLoad() {
        super.viewDidLoad()
        // Do any additional setup after loading the view.
    }
    
    override func touchesBegan(_ touches: Set<UITouch>, with event: UIEvent?) {
        for i in 0..<2 {
            print("......\(i)")
            sleep(1)
        }
    }
    
}

